crates/iota-rosetta/tests/end_to_end_tests.rs

```diff
@@ -206,17 +201,15 @@ async fn test_stake_all() {
 
     let (rosetta_client, _handle) = start_rosetta_test_server(client.clone()).await;
 
-    let system_state = client
+    let validator = client
```

Comment:
```suggestion
    let committee_member_address = client
```

Reply:
Thanks, addressed it in [53586c7](https://github.com/iotaledger/iota/pull/6015/commits/53586c7a3ac537750ba4f200b7ba5fc506454779).